crates/compiler/builtins/bitcode/src/list.zig

```diff
@@ -740,9 +740,8 @@ fn swapElements(source_ptr: [*]u8, element_width: usize, index_1: usize, index_2
 }
 
 pub fn listConcat(list_a: RocList, list_b: RocList, alignment: u32, element_width: usize) callconv(.C) RocList {
-    if (list_a.isEmpty()) {
-        return list_b;
-    } else if (list_b.isEmpty()) {
+    // NOTE we always use list_a! because it is owned, we must consume it, and it may have unused capacity
```

Comment:
list_b is also owned in the borrow signature, so should we decref it here if empty?

Reply:
yes, fixed! 

I think this is from when we had no capacity, and an empty list would just not have an allocation.